content/participants/ttjc@SUSMOY97/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "SUSMOYOP"
+type: "participant"
+name: "SUSMOY MUKHERJEE"
+portfolio: "https://tanaypratap.com"
+github-link: "https://github.com/SUSMOY97"
+linkedin-link: "www.linkedin.com/in/susmoymukherjee"
+looking-for: "job"
+description: "expert at CSS/HTML learning MERN stack"
+---
+
+Hi! I am Susmoy Mukherjee a self made web developer currently developing skills in MERN stack.
+
+### introduction
+
+I am a undergrad highly interested in React and it's awesome UI libraries and playing with it by making different projects .
+
+## why are you taking this challenge?
+
+I need a job as well as due to this challenge my resume is getting updated with more projects and i am learning a lot.
+
+## something interesting about you?
+
+I can eat 3 packets of Biriyani at a time XD.
+
+### help I need?
+
+I am making various projects for this challenge on react as well as learning react everyday , so i might need help with react-router.
+
+### how to contact me?
+
+- Twitter
```

Comment:
provide a link to your social handle or remove this. also suggestion use markdown to sinplify things

Reply:
Done